ocs_ci/ocs/platform_nodes.py

```diff
@@ -2420,3 +2420,63 @@ def restart_nodes_by_stop_and_start_teardown(self):
                 cmd = f"ibmcloud ks worker reboot --cluster {cluster_id} --worker {not_ready_node} -f"
                 out = run_cmd(cmd)
                 logger.info(f"Node restart command output: {out}")
+
+    def create_nodes(self, node_conf, node_type, num_nodes):
+        """
+        Create new new node by the end of the test on IBM Cloud.
+
+        Args:
+            node_conf (dict): of node configuration
+            node_type (str): type of node to be created RHEL
+            num_nodes (int): Number of node instances to be created
+
+        Returns:
+           list: of IBMCloudNode objects
+
+        """
+        logger.info("creating new node")
+        login()
+
+        worker_nodes = get_nodes(node_type="worker")
+        provider_id = worker_nodes[0].get()["spec"]["providerID"]
+        cluster_id = provider_id.split("/")[5]
+
+        cmd = f"ibmcloud ks workers --cluster {cluster_id} --output json"
+        out = run_cmd(cmd)
+        worker_nodes = json.loads(out)
+
+        no_of_nodes = len(worker_nodes) + int(num_nodes)
+        logger.info(f"number of nodes going to be add are : {no_of_nodes}")
+
+        cmd = (
+            f"ibmcloud ks worker-pool resize --cluster {cluster_id} --worker-pool default"
+            f"  --size-per-zone {no_of_nodes}"
+        )
+        out = run_cmd(cmd)
```

Comment:
If we are not using out, we can use run_cmd(cmd)

Reply:
we are using out parameter here
worker_nodes = json.loads(out)

Reply:
could you point the line where we are using ? did I miss something here